examples/kitchen-sink/apps/storefront/package.json

```diff
@@ -4,10 +4,10 @@
   "private": true,
   "scripts": {
     "build": "next build",
-    "start": "next start ",
+    "clean": "rm -rf .turbo && rm -rf node_modules && rm -rf .next",
```

Comment:
node_modules is no-op, .turbo we can leave alone, so ... just `.next`?

Reply:
Yep you're right, I just keep forgetting to remove these 😂